image-hotspot-question.js

```diff
@@ -381,6 +382,17 @@ H5P.ImageHotspotQuestion = (function ($, Question) {
     $.extend(true, definition, this.getxAPIDefinition());
   };
 
+  /**
+   * Add the response part to an xAPI event
+   *
+   * @param {H5P.XAPIEvent} xAPIEvent
+   *  The xAPI event we will add a response to
+   */
+  ImageHotspotQuestion.prototype.addResponseToXAPI = function (xAPIEvent) {
+    var self = this;
+    xAPIEvent.setScoredResult(self.getScore(), self.getMaxScore(), self, true, true);
```

Comment:
I think the "succeeded" flag should be set based on the actual score, cmp PR at https://github.com/h5p/h5p-image-hotspot-question/pull/70 and comment at https://github.com/h5p/h5p-image-hotspot-question/commit/7368e45794b7ca3dac1f69d260c4720e70931b5c#r75501071